packages/trie/src/trie/secure.ts

```diff
@@ -119,7 +119,7 @@ export class SecureTrie extends CheckpointTrie {
    * Persists the root hash in the underlying database
    */
   async persistRoot() {
-    if (this._persistRoot !== undefined) {
+    if (this._persistRoot === true) {
```

Comment:
Sorry, side topic: with the linting I would actually personally prefer if we would allow `if (this._persistRoot)` or similar for all boolean values (since there it is just a binary choice anyhow and the short version is just more elegant, and only enforce an explicit comparison 2nd value for all non-boolean values.

Would this be possible? 🤔 //cc @gabrocheleau

Reply:
Ah ok, that's true. Meanwhile often forget about JS completely. 😬 (which is also not that good, we should really - also - keep to have the JS users in mind, so in the sense of JS Developers, and at the end every *end* user is a JS user anyhow).